{"uuid": "a2a4be16-e893-4985-a683-f38e1fd50589", "children": ["4e2d9ce6-7ab2-4f68-8a68-55f31690f87e", "26f9d3f8-cce8-444c-963e-bff2f22d231c"], "befores": [{"name": "login_token", "status": "skipped", "statusDetails": {"message": "Skipped: ❌ 登录失败，跳过后续测试\n", "trace": "  File \"D:\\桌面\\pytest_demo\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"D:\\桌面\\pytest_demo\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"D:\\桌面\\pytest_demo\\venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"D:\\桌面\\pytest_demo\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\桌面\\pytest_demo\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1196, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\桌面\\pytest_demo\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 930, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"D:\\桌面\\pytest_demo\\conftest.py\", line 28, in login_token\n    pytest.skip(\"❌ 登录失败，跳过后续测试\")\n    ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\桌面\\pytest_demo\\venv\\Lib\\site-packages\\_pytest\\outcomes.py\", line 159, in skip\n    raise Skipped(msg=reason, allow_module_level=allow_module_level)\n"}, "start": 1760703684187, "stop": 1760703685895}], "start": 1760703684187, "stop": 1760703688663}